lib/helper.js

```diff
@@ -60,8 +60,12 @@ class Helper {
    * @return {string}
    */
   static getExceptionMessage(exceptionDetails) {
-    if (exceptionDetails.exception)
+    if (exceptionDetails.exception) {
+      if (exceptionDetails.exception.type === 'string') {
```

Comment:
let's return `description` if there's any, and fallback to `value` otherwise.
This way we'll support throwing numbers as well